src/View/User/index.phtml

```diff
@@ -2,12 +2,14 @@
 
 use PHPCensor\Helper\Lang;
 
-$user = $this->getUser();
+$currentUser = $this->getUser();
 
 ?>
 <div class="clearfix"  style="margin-bottom: 20px;">
     <div class="pull-right btn-group">
+        <?php if($currentUser->getIsAdmin()): ?>
```

Comment:
This condition is useless because you already check admin rights in the controller.

Reply:
I guess you should remove `$this->requireAdmin();` from controller action `index`.

Reply:
Yeah I was a bit in too minds about this, I had already made the view non-admin user friendly before I added the requireAdmin in the controller. Thought I would leave this in incase someone wanted to allow users to view that page. 

Do you have a preference? If we allow users to view the /user action then we'll also need to clean up the edit view so that users can't promote themselves to admin.

Reply:
Removing  `$this->requireAdmin();` from list action would be good solution for me for now.

Reply:
I've updated so that users can see the listing and edit their own account. Admins can ofcourse edit everyone :)

Reply:
Thanks!